ci/deps/pyspark.yml

```diff
@@ -1 +1,3 @@
-pyspark>=2.4.3
+double-conversion  # Otherwise `import pyarrow` fails with ImportError: libdouble-conversion.so.3
```

Comment:
https://github.com/dask/dask-docker/issues/98

Seems like newer pyarrow doesn't have this issue?

Reply:
If so, we should probably note this can be moved when upgrading to newer pyarrow version

Reply:
Yep I believe at newer versions of `pyarrow`, I didn't need to add this dependency. I'll add a note to the comment here